src/models/maze.c

```diff
@@ -79,17 +112,17 @@ void freeMaze()
     free2DArray(gameMaze, MAP_HEIGHT);
 }
 
-bool isObstacle(struct Position position)
+bool isPositionContainsMazeElement(struct Position position, MazeElement mazeElementToCheck)
```

Comment:
I'm not sure if is useful to have a function that do a == comparaison.
I think that the code can be more readable if we do this comparaison directly where we need in than in a function.
See comment in pacman.c

Reply:
[11aacd3](https://github.com/Livl-Corporation/livl-pacman/commit/11aacd36abf4138cba46cbf5e638747a83c9ee9c)